tree/dataframe/src/RLoopManager.cxx

```diff
@@ -595,110 +560,6 @@ bool validTTreeReaderRead(TTreeReader &treeReader)
 }
 } // namespace
 
-/// Run event loop over one or multiple ROOT files, in parallel.
```

Comment:
Ubuntu 22 is issuing a warning that the function above is unused.

Reply:
Thanks! That wasn't caught on my machine

Reply:
Also found out why, it's because the function is now only used in the MT case, and the method calling it is guarded by `#ifdef R__USE_IMT`, adding the guard also to this function.